test/dlc_tests/container_tests/bin/pytorch_tests/testCurand

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if ls /usr/local/cuda/include | grep curand; then
```

Comment:
is it a file called curand that we're looking for, or curand.h? are there multiple files and is there a way we can be deliberate about looking for them?

Reply:
Yes, It is looking for a particular package known as curand.h. I will grep curand.h instead of curand in the next iteration.